devenv.nix

```diff
@@ -141,35 +143,41 @@ let
     TARGETFOLDER="${config.env.DEVENV_STATE}/importdb"
 
     rm -rf "$TARGETFOLDER"
-
     set -e
 
     if [[ "$1" == *.sql ]]; then
-        curl --create-dirs "$1" --output "$TARGETFOLDER/dump.sql"
-    elif [[ "$1" == *.sql.gz ]]; then
-        curl --create-dirs "$1" --output "$TARGETFOLDER/latest.sql.gz"
-        gunzip -c "$TARGETFOLDER/latest.sql.gz" > "$TARGETFOLDER/dump.sql"
-    elif [[ "$1" == *.sql.zip ]]; then
-        curl --create-dirs "$1" --output "$TARGETFOLDER/latest.sql.zip"
-        unzip -j -o "$TARGETFOLDER/latest.sql.zip" '*.sql' -d "$TARGETFOLDER"
+      ${pkgs.curl}/bin/curl --create-dirs "$1" --output "$TARGETFOLDER/dump.sql"
+    elif [[ "$1" == *.gz ]]; then
+      ${pkgs.curl}/bin/curl --create-dirs "$1" --output "$TARGETFOLDER/latest.sql.gz"
+      ${pkgs.gzip}/bin/gunzip -c "$TARGETFOLDER/latest.sql.gz" > "$TARGETFOLDER/dump.sql"
+    elif [[ "$1" == *.zip ]]; then
+      ${pkgs.curl}/bin/curl --create-dirs "$1" --output "$TARGETFOLDER/latest.sql.zip"
+
+      if [ x"$IMPORT_DATABASE_PASSWORD" == "x" ]; then
+        UNZIP_COMMAND="${pkgs.unzip}/bin/unzip"
+      else
+        UNZIP_COMMAND="${pkgs.unzip}/bin/unzip -P $IMPORT_DATABASE_PASSWORD"
```

Comment:
I'm not sure if we should add this functionality. It is prone to user error (committing passwords) and brittle (special characters breaking shell expansion).
Is this a must-have for you?

Reply:
It would streamline some processes, ive removed it for now for a more integrated solution.